kirby/web/endpoints.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+import copy
+import sqlalchemy
+from flask import abort
+from flask_restplus import Api, Resource, fields
+from ..models import db, Job
+
+api = Api()
+
+group_model = api.model("Group", {"name": fields.String})
+
+notifications_model = api.model(
+    "Notifications",
+    {
+        "on_retry": fields.Boolean,
+        "on_failure": fields.Boolean,
+        "groups": fields.List(fields.Nested(group_model)),
+    },
+)
+
+job_model = api.model(
+    "Job",
+    {
+        "name": fields.String,
+        "package_name": fields.String,
+        "package_version": fields.String,
+        "notifications": fields.List(fields.Nested(notifications_model)),
+    },
+)
+
+schedule_model = api.model(
+    "Schedule",
+    {
+        "date": fields.String(default=str(datetime.utcnow())),
+        "jobs": fields.List(fields.Nested(job_model)),
+    },
+)
+
+
+@api.route("/schedule")
+class Schedule(Resource):
+    @api.marshal_with(schedule_model)
+    def get(self):
+        date = datetime.utcnow()
+        jobs = []
+        schedule = {"date": date, "jobs": jobs}
+
+        for job in db.session.query(Job).all():
+            job_name = job.name
+
+            # Get Notifications data
+            notifications = []
+            for notification in job.notifications:
+                on_retry = notification.on_retry
+                on_failure = notification.on_failure
+                groups = [
+                    {"name": group.name} for group in notification.groups
```

Comment:
I would add also the notification methods in this, so we don't have to look it up in case of problem

Reply:
You mean `group.emails` ?  
There is no more information on the `groups` for now

Reply:
yep, please add the emails from the group notification emails

Reply:
Ok, that is now added